test/HydrationRepository-test.js

```diff
@@ -2,6 +2,7 @@ const chai = require('chai');
 const expect = chai.expect;
 
 import HydrationRepository from '../src/HydrationRepository';
+import UtilityRepository from '../src/UtilityRepository';
```

Comment:
Looking at this locally on my machine, it seems like we aren't using the UtilityRepo anywhere other than this import line - can you clarify what it's purpose is here?

Reply:
It needed the UtilityRepository import to access the inherited methods from it.  Or, at least that seems to have been one of the issues. @sertmer